v2/support/modal_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,88 @@
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+  <section data-main>Loading...</section>
+  <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    // https://developer.zendesk.com/apps/docs/apps-v2
+    var client = ZAFClient.init();
+
+    function init(location) {
+      location === 'modal' ? new ModalApp() : new TicketApp();
```

Comment:
I wrote it like this [in the scaffold](https://github.com/zendesk/app_scaffold/blob/master/src/javascripts/index.js#L10) to simplify development, so that developers could focus their attention on the individual app classes and not have to worry about the index loading code. So ideally the loading code wouldn't have to know anything about the app class. We could perhaps have a common `init` method that gets called instead of `render`? I do see the point of not having side-effects in constructors, but in this case there's no interaction between objects, the app is simply reacting to events, so it's not really an issue here.